src/bp/core/services/nlu/intent-service.ts

```diff
@@ -67,51 +70,19 @@ export class IntentService {
         }
 
         for (let i = 0; i < explicitIntents.length; i++) {
-          const intentName = sanitizeFileName(`${flow.name}/${tn?.name}/${conditions.length + i}`)
+          const intentName = sanitizeFileName(`${flow.name}/${triggerNode?.name}/${conditions.length + i}`)
           if (intentsByName[intentName]) {
             throw new Error(`Duplicated intent with name "${intentName}"`)
           }
           intentsByName[intentName] = {
-            contexts: [sanitizeFileName(`explicit:${flow.name}/${node.name}`)],
+            contexts: [sanitizeFileName(`explicit:${flow.name}/${triggerNode.name}`)],
             filename: flow.name,
             name: intentName,
             slots,
             utterances: explicitIntents[i]?.params?.utterances ?? {}
           }
         }
       }
-
-      // for (const node of flow.nodes.filter(x => x.type === 'say_something')) {
-      //   let idx = 0
-      //   for (let i = 0; i < (node.contents?.length ?? 0); i++) {
-      //     node.contents![i].choices?.forEach((choice, c) => {
-      //       const utterances = [choice.title, choice.value].filter(Boolean).reduce((utterances, obj) => {
-      //         // TODO: convert this once we have the GUI implemented
-      //         for (const lang of Object.keys(obj)) {
-      //           if (!utterances[lang]) {
-      //             utterances[lang] = []
-      //           }
-      //           utterances[lang] = [...utterances[lang], obj[lang]]
-      //         }
-
-      //         return utterances
-      //       }, {})
-
-      //       if (Object.keys(utterances)) {
-      //         const intentName = sanitizeFileName(`${flow.name}/${node.name}/${idx++}`) // TODO: change this for user_isdetection
-      //         const contextName = sanitizeFileName(`explicit:${flow.name}/${node.name}`)
-      //         intentsByName[intentName] = {
-      //           contexts: [contextName],
-      //           filename: flow.name,
-      //           name: intentName,
-      //           slots: [],
-      //           utterances: utterances
-      //         }
-      //       }
-      //     })
-      //   }
-      // }
-      //
```

Comment:
Won't removing this break suggestions?

Reply:
that's dead code ...  There is no need for commented code